compiler/rustc_llvm/src/lib.rs

```diff
@@ -90,6 +90,13 @@ pub fn initialize_available_targets() {
         LLVMInitializeAVRAsmPrinter,
         LLVMInitializeAVRAsmParser
     );
+    init_target!(
+        llvm_component = "m68k",
+        LLVMInitializeM68kTargetInfo,
+        LLVMInitializeM68kTarget,
+        LLVMInitializeM68kTargetMC,
+        LLVMInitializeM68kAsmPrinter
+    );
```

Comment:
```suggestion
        LLVMInitializeM68kAsmPrinter,
        LLVMInitializeM68kAsmParser
    );
```

I think this is needed for inline assembly (which should work in LLVM 13).

Reply:
Ah, right. When I first worked on those patches, we didn't have the parser yet. Let me try.